plotnine/scales/scale_color.py

```diff
@@ -454,8 +454,8 @@ class scale_color_cmap(scale_continuous):
     cmap_name :
         A standard Matplotlib colormap name. The default is
         `viridis`. For the list of names checkout the output
-        of `matplotlib.cm.cmap_d.keys()` or see the
-        `documentation <http://matplotlib.org/users/colormaps.html>`_.
+        of `matplotlib.colormaps()` or see the
+        [documentation](http://matplotlib.org/users/colormaps.html).
```

Comment:
The current link redirects to https://matplotlib.org/2.0.2/users/colormaps.html which shows a red banner "You are reading an old version of the documentation (v2.0.2). For the latest version see https://matplotlib.org/stable/"

I think the new link should be:

```suggestion
        [documentation](https://matplotlib.org/stable/users/explain/colors/colormaps.html).
```